tests/data_files/Makefile

```diff
@@ -1048,6 +1048,99 @@ cert_md5.crt: cert_md5.csr
 	$(MBEDTLS_CERT_WRITE) request_file=$< serial=6 issuer_crt=$(test_ca_crt) issuer_key=$(test_ca_key_file_rsa) issuer_pwd=$(test_ca_pwd_rsa) not_before=20190210144406 not_after=20290210144406 md=MD5 version=3 output_file=$@
 all_final += cert_md5.crt
 
+#PKCS7 test data
+pkcs7_test_cert_1 = pkcs7-rsa-sha256-1.crt
+pkcs7_test_cert_2 = pkcs7-rsa-sha256-2.crt
+pkcs7_test_file = pkcs7_data.txt
+
+#Generate signing cert
+pkcs7-rsa-sha256-1.crt:
+	$(OPENSSL) req -x509 -subj="/C=NL/O=PKCS7/CN=PKCS7 Cert 1" -sha256 -nodes -days 365  -newkey rsa:2048 -keyout pkcs7-rsa-sha256-1.key -out pkcs7-rsa-sha256-1.crt
+	cat pkcs7-rsa-sha256-1.crt > pkcs7-rsa-sha256-1.pem
+	cat pkcs7-rsa-sha256-1.key >> pkcs7-rsa-sha256-1.pem
```

Comment:
you can write `cat pkcs7-rsa-sha256-1.crt  pkcs7-rsa-sha256-1.key > pkcs7-rsa-sha256-1.pem` as cat will automatically con**cat**enate its arguments :)